dist/justified-layout.js

```diff
@@ -45,8 +45,8 @@ var Row = module.exports = function (params) {
 	this.maxAspectRatio = this.width / params.targetRowHeight * (1 + params.targetRowHeightTolerance);
 
 	// Edge case row height minimum/maximum
-	this.edgeCaseMinRowHeight = params.edgeCaseMinRowHeight || Number.NEGATIVE_INFINITY;
-	this.edgeCaseMaxRowHeight = params.edgeCaseMaxRowHeight || Number.POSITIVE_INFINITY;
```

Comment:
Looks like both of these things were removed in 3159e445ec67e32c05b72bc040428fef2ee7ad28 and the built file was never checked in. 😕 

Definitely agree that this file shouldn't even be checked in. After this lands I'll make it part of the build.